modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v8_8_2/SchemaUpgradeProcess.java

```diff
@@ -24,8 +25,13 @@ protected void doUpgrade() throws Exception {
 				SQLTransformer.transform(
 					StringBundler.concat(
 						"select ObjectField.dbColumnName, ",
-						"ObjectField.dbTableName from ObjectField where ",
-						"ObjectField.businessType ='",
+						"ObjectField.dbTableName from ObjectField inner join ",
+						"ObjectDefinition on ",
+						"ObjectDefinition.objectDefinitionId = ",
+						"ObjectField.objectDefinitionId where ",
+						"ObjectDefinition.status != '",
+						WorkflowConstants.STATUS_DRAFT,
```

Comment:
I think if we `put = WorkflowConstants.STATUS_APPROVED` it would be easier to understand the query.

Reply:
Since the status is a `int`, we don't need to put `' '`.

Reply:
Please, test the query locally.

Reply:
example: ArticleExpirationDateUpgradeProcess.java

Reply:
Done. Thanks.